spec/models/user_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+    TodoList.new(title: "Anything", description: "Lorem ipsum",

```

Comment:
You could add FactoryBot for your tests

Reply:
thanks for your comment, but in some case it was not necesary, but for many tests i will concider, thaks ;) :D